tests/features/secure-timeseries.feature

```diff
@@ -98,6 +104,11 @@ Feature: Vermillion is able to handle secure timeseries datasets
            When The consumer requests for a standalone authorised ID with invalid token
            Then The response status should be 403
 
+         Scenario: Authorised ID - single
```

Comment:
The description is misleading. The title is authorised ID, but the test is checking with an unauthorised ID

Reply:
Made changes